app/models/likes.rb

```diff
@@ -1,4 +1,8 @@
 class Like < ActiveRecord::Base
-  belong_to :user
-  belong_to post
+  belongs_to :user, foreign_key: :author_id
+  belongs_to post, foreign_key: :post_id
+
+  def update_likes_conuter
+    post.increment!(:likes_counter)
+  end
```

Comment:
- [ ] Let's use Rails Active Callback `after_save` to call `update_likes_conuter` method.